README.md

```diff
@@ -89,6 +89,145 @@ refresh_token = jwt_response[REFRESH_SESSION_TOKEN_NAME].get("jwt")
 
 The session and refresh JWTs should be returned to the caller, and passed with every request in the session. Read more on [session validation](#session-validation)
 
+### Enchanted Link
+
+Send an email containing three numbered links to a user.
+This method is similar to [Magic Link](#magic-link) but differs in two major ways:
+
+- The user must choose the correct link according to a number that's
+  displayed on the web page they're trying to log into
+- This supports cross-device clicking, meaning the user can try to log in on one device,
+  like a computer, while clicking the link on another device, for instance a mobile phone.
+
+The Enchanted Link will redirect the user to page where the its token needs to be verified.
+This redirection can be configured in code, or generally in the [Descope Console](https://app.descope.com/settings/authentication/enchantedlink)
+
+The user can either `sign up`, `sign in` or `sign up or in`
+
+```python
+resp = descope_client.enchantedlink.sign_up_or_in(
+    identifier=email,
+    uri="http://myapp.com/verify-magic-link", # Set redirect URI here or via console
+)
+link_identifier = resp["linkId"] # Show the user which link they should press in their email
+pending_ref = resp["pendingRef"] # Used to poll for a valid session
```

Comment:
no consts for "pendingRef" and "linkId" ?

Reply:
@guyp-descope ?

Reply:
Nop. we dont have. I think we can keep it like that..